docs/CHANGELOG.asciidoc

```diff
@@ -13,12 +13,553 @@
+Removes the backwards compatibility logic that checked for duplicative settings 
+`map.regionmap` and `map.tilemap`. {pull}28609[#28609]
+Monitoring::
+* Refactors index patterns from configurable to constants {pull}29528[#29528]
+* Removes `node_resolver` setting {pull}21181[#21181]
+Management::
+* Adds `configPrefix` to Index Management, License Management, Rollup Jobs, and 
+Upgrade Assistant {pull}30149[#30149]
+* Removes nested table splits from table visualization {pull}26057[#26057]
+* Standardizes use of `index_pattern` in UI code and URLs, replacing `indices` 
+and `index`  {pull}29151[#29151] and {pull}29481[#29481]
+* Removes app-specific dark mode options and replaces with a global dark mode uiSetting {pull}29017[#29017]
+Operations::
+* Defers the optimize step in the plugin installer to server start {pull}26983[#26983]
+* Removes tribe node support {pull}16397[#16397]
+* Creates separate startup scripts for development and production {pull}13806[#13806]
+* Sets default port based on protocol  {pull}21564[#21564]
+* Removes deprecated `/shorten` API {pull}21861[#21861]
+* Plugin installer defers optimization step until server start {pull}26983[#26983]
+Platform::
+* Removes deprecated `i18n.defaultLocale` setting {pull}29325[#29325]
+* Removes previously deprecated `elasticsearch*` settings. {pull}29496[#29496]
+* Adds new "references" attribute to saved objects for relationships {pull}28199[#28199]
+Querying & Filtering::
+* Makes KQL the default query language for new searches {pull}27092[#27092]
+Reporting::
+* Removes Phantom dependency {pull}27142[#27142]
+Security::
+* Removes deprecated SSL settings {pull}28622[#28622]
+* Removes legacy fallback {pull}29107[#29107]
+Visualizations::
+* Removes Timelion from the app switcher by default {pull}30131[#30131]
+
+
+[float]
+[[deprecation-7.0.0-rc1]]
+=== Deprecations
+
+Geo::
+* Fixes legacy tilemap loading {pull}22095[#22095]
+Machine Learning::
+* Updates REST endpoint calls to remove _xpack {pull}28069[#28069]
+Visualizations::
+* Removes old visualize wizard URL {pull}28982[#28982]
+* Removes legacy response handler {pull}28985[#28985]
+* Removes `legacyIcon` from visualizations {pull}28981[#28981]
+
+[float]
+[[highlight-7.0.0-rc1]]
+=== Highlights
+Canvas::
+* Canvas now has a template tab in the workpad manager where users can find workpad templates, 
+demos, and tutorials to help them get started. See {pull}23966[#23966] for more information.
+Kibana App:: 
+* Visualizations in Kibana will use a new data pipeline introduced as part of Canvas. 
+The change does not yet apply to Vega, Timelion, or Time Series Visual Builder (TSVB). 
+See {pull}25711[#25711] for more information.
+Index Lifecyle Management:: 
+* Implements a user interface to create, update, edit, or delete index lifecyle policies.
+See {pull}25553[#25553] for more information.
+K7 UI Design::
+* {kib} 7.0.0-rc1 includes a new design for {kib} called K7. In this early stage, 
```

Comment:
You cannot switch this off any more. We should remove this paragraph and add a note about K7 design to the Breaking changes section.

Reply:
If you add to the Breaking changes section, then it should also be included in the Breaking changes doc.

Reply:
Okay, I will move to breaking changes and check with Design about the wording.